src/aui/tabart.cpp

```diff
@@ -164,12 +164,35 @@ wxAuiGenericTabArt::wxAuiGenericTabArt()
     m_fixedTabWidth = wxWindow::FromDIP(100, NULL);
     m_tabCtrlHeight = 0;
 
+    SysColourChanged();
+
+    m_activeCloseBmp = wxAuiBitmapFromBits(close_bits, 16, 16, *wxBLACK);
+    m_disabledCloseBmp = wxAuiBitmapFromBits(close_bits, 16, 16, wxColour(128,128,128));
+
+    m_activeLeftBmp = wxAuiBitmapFromBits(left_bits, 16, 16, *wxBLACK);
+    m_disabledLeftBmp = wxAuiBitmapFromBits(left_bits, 16, 16, wxColour(128,128,128));
+
+    m_activeRightBmp = wxAuiBitmapFromBits(right_bits, 16, 16, *wxBLACK);
+    m_disabledRightBmp = wxAuiBitmapFromBits(right_bits, 16, 16, wxColour(128,128,128));
+
+    m_activeWindowListBmp = wxAuiBitmapFromBits(list_bits, 16, 16, *wxBLACK);
+    m_disabledWindowListBmp = wxAuiBitmapFromBits(list_bits, 16, 16, wxColour(128,128,128));
+
+    m_flags = 0;
+}
+
+wxAuiGenericTabArt::~wxAuiGenericTabArt()
+{
+}
+
+void wxAuiGenericTabArt::SysColourChanged()
+{
 #if defined( __WXMAC__ ) && wxOSX_USE_COCOA_OR_CARBON
     wxColor baseColour = wxSystemSettings::GetColour(wxSYS_COLOUR_WINDOW);
 #else
     wxColor baseColour = wxSystemSettings::GetColour(wxSYS_COLOUR_3DFACE);
 #endif
-
+    
```

Comment:
Please try to avoid adding extra whitespace, it's minor but annoying.

Putting `exec git diff-index --check --cached HEAD --` in your `.git/hooks/pre-commit` and doing `git config core.whitespace blank-at-eol,space-before-tab,cr-at-eol` will prevent you from committing such whitespace additions accidentally.